package.json

```diff
@@ -193,7 +194,7 @@
     "react": "16.x",
     "react-dom": "16.x",
     "react-intl": "2.x",
-    "react-router-dom": "4.x"
+    "react-router-dom": "4.x || 5.x"
```

Comment:
we are already using 5 in our apps

Reply:
In a perfectly fine semver world this would be a breaking change too. As we change our peer deps and the consumer does have to change something (even though it's not a breaking change in itself). However, we also didn't bump to breaking in app-kit.

Reply:
We should just go with `5.x`